### PR TITLE
George branch

### PR DESCRIPTION
**Pull request by georgehuliaris**

### CODE REVIEW

**Review by georgehuliaris — COMMENTED** [review]
looks good

File: README.md [hunk 1]
```diff
@@ -1,2 +1,32 @@
 # missedOpts
-hey hey hey
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (georgehuliaris) [inline README.md:2]: looks good!

**Review by georgehuliaris — COMMENTED** [review]
I like it

**Comment by georgehuliaris** [comment]
I like it

**Review by georgehuliaris — COMMENTED** [review]
is this ok?

**Review by alligatoraustin — APPROVED** [review]

**Pull request merged**